src/pytorch_lightning/trainer/connectors/logger_connector/result.py

```diff
@@ -30,6 +30,7 @@
 from pytorch_lightning.utilities.memory import recursive_detach
```

Comment:
Why are there changes in this file? I don't see a connection to the saving utils.

Reply:
reverted to https://github.com/Lightning-AI/lightning/pull/13932/commits/0502a4e9fe2f1b229265de8ae82a129e704f65ee and synced with upstream in https://github.com/Lightning-AI/lightning/pull/13932/commits/82fc4528431fd3412e30782ba64bcc65d2adba38